src/models/StandaloneSimilarity.py

```diff
@@ -1,27 +1,169 @@
-"""
-import random
+import gc
+import itertools
+import math
+from multiprocessing.pool import Pool
 
-from cleaning_functions import pandas_vector_to_list
-from job_description_feature_extraction import get_tfidf_similarity, extract_relevant_documents
+import numpy as np
+import os
+from sklearn.feature_extraction.text import TfidfVectorizer
+from tqdm import tqdm
 
-"""
+from cleaning_functions import pandas_vector_to_list
+from job_description_feature_extraction import (tfidf_vectorize,
+                                                cosine_knn, calc_cosine_sim,
+                                                extract_top_k_documents)
 from models.BaseModel import BaseModel
 
+DATA_QUERIES_VECTOR_NPZ = '../preprocessing_data/queries_vector.npz'
+DATA_CORPUS_VECTOR_NPZ = '../preprocessing_data/corpus_vector.npz'
+DATA_SIMILIARITY_MATRIX = '../preprocessing_data/similiarity_matrix'
```

Comment:
npz is a numpy format
see https://docs.scipy.org/doc/numpy/reference/generated/numpy.savez.html

its very much used in models.StandaloneSimilarity.StandaloneSimilarity#store_precomputed_data